content/docs/projects/trust-manager/README.md

```diff
@@ -56,79 +76,293 @@ spec:
+
+```bash
+kubectl --kubeconfig ./bin/kubeconfig.yaml apply -f - <<EOF
+apiVersion: trust.cert-manager.io/v1alpha1
+kind: Bundle
+metadata:
+  name: example-bundle
+spec:
+  sources:
+  - useDefaultCAs: true
+  - secret:
+      name: "trust-manager-example-ca-secret"
+      key: "tls.crt"
+  target:
+    configMap:
+      key: "trust-bundle.pem"
+EOF
 ```
 
+And we're done! The `example-bundle` `ConfigMap` should already be updated.
+
+If you inspect the `ConfigMap` again, the last certificate you'll see in the list
+should be our new dummy CA.
+
+## Securely Maintaining A trust-manager Installation
+
+If you choose the `useDefaultCAs` source on any of your `Bundle` resources, it's important that you keep your
+default CA package image up to date. Failing to do so would be the equivalent of failing to run `apt-get upgrade ca-certificates`
+when installing a public trust bundle in a Debian container.
+
+trust-manager has been designed in such a way that any version of any default CA package should work with any
+version of trust-manager which supports default CAs (`v0.4.0` and later). There should be no risk to the stability
+of trust-manager from upgrading.
+
+If you're using an official cert-manager-provided Debian CA package (which is the default), you should check which version you have
+and compare against the [latest package version](https://quay.io/repository/jetstack/cert-manager-package-debian?tab=tags&tag=latest).
+
+The version can be configured with the `.defaultPackageImage.tag` value on the Helm chart, and the version
+is also written to the `status` field on any synced Bundle resource which uses the default CA package.
+
+## Upgrading A Default CA Package Using Helm
+
+Say we want to do an in-place upgrade of our default CA package to tagged version `XYZ` - without upgrading trust-manager.
+
+We'll assume the Helm release is called "trust-manager" and that we've installed into the `cert-manager` namespace.
+
+> ⚠️ This upgrade process assumes that it's the only thing running. If another user or process changes Helm values
+> while you're doing this process, you might overwrite their work.
+
+First, we'll dump our current Helm values, so we don't lose them:
+
 ```bash
-kubectl get cm -A --field-selector=metadata.name=example-bundle
-NAMESPACE            NAME             DATA   AGE
-cert-manager         example-bundle   1      2m18s
-default              example-bundle   1      2m18s
-kube-node-lease      example-bundle   1      2m18s
-kube-public          example-bundle   1      2m18s
-kube-system          example-bundle   1      2m18s
-local-path-storage   example-bundle   1      2m18s
+helm get values -n cert-manager trust-manager -oyaml > values.yaml
+```
+
+Next, if `defaultPackageImage.tag` is already set in `values.yaml`, update it. Otherwise, add it.
+You can find the available tags [on `quay.io`](https://quay.io/repository/jetstack/cert-manager-package-debian?tab=tags&tag=latest).
```

Comment:
I looked at that page and was confused about why the tag is a date in 2021 but the  image appears to be last modified only 18 days ago.

![image](https://user-images.githubusercontent.com/978965/216599209-ac7db19d-74f2-4ee2-b632-c4f8713d82e6.png)


I suggest explaining the significance of the date in these tags.

Reply:
Good idea, yeah. I'm struggling for time today but I'll try to get this done!